go.mod

```diff
@@ -404,3 +405,7 @@ replace (
 
 // exclusion so we use v1.0.0
 exclude github.com/coinbase/rosetta-sdk-go v0.7.9
+
+exclude github.com/cosmos/cosmos-sdk v0.50.1
+
+exclude github.com/cometbft/cometbft v0.38.0
```

Comment:
These keep auto-updating on `go mod tidy`. That breaks proto-generation CI.

As a result, I manually excluded them here